agentops/state.py

```diff
@@ -0,0 +1,9 @@
+state_dict = {}
+
+
+def set_state(key: str, value: any):
+    state_dict[key] = value
+
+
+def get_state(key: str) -> any:
+    return state_dict.get(key)
```

Comment:
Why isn't state just a part of client?

Reply:
because if the dev never calls `.init()`, we dont even want to create a client. essentially this blocks _everything_ if init is never called

Reply:
And for `is_initialized`, instead of adding a decorator to every function, I think the functions themselves should just check 
```
if Client().is_initialized:
  ...
```